lib/_http_outgoing.js

```diff
@@ -78,11 +78,12 @@ exports.OutgoingMessage = OutgoingMessage;
 OutgoingMessage.prototype.setTimeout = function(msecs, callback) {
   if (callback)
     this.on('timeout', callback);
-  if (!this.socket) {
+
+  if (!this.socket)
```

Comment:
I actually do agree with you but I changed it to match the function declaration just a few lines below. I'll update with a commit that fixes this one and the one below, too.